<launch>
  <node pkg="nodelet" type="nodelet" name="nodelet_manager" args="manager" />
  <node pkg="nodelet" type="nodelet" name="nodelet1"
    args="load depth_image_proc/point_cloud_xyz nodelet_manager">
    <remap from="camera_info" to="/object_detection/camera_info" />
    <remap from="image_rect" to="/object_detection/object_depth" />
    <remap from="points" to="/object_detection/points" />
  </node>

  <node pkg="final_assignment" type="object_detection.py" name="object_detection" output="screen">
    <!-- <remap from="/camera/rgb/image_rect_color" to="/camera/rgb/image_raw" />
    <remap from="/camera/depth_registered/image_raw" to="/camera/depth/image_raw" />
    <remap from="/camera/depth_registered/camera_info" to="/camera/depth/camera_info" /> -->
  </node>
  <node pkg="final_assignment" type="object_coord.py" name="object_coord" output="screen" />
</launch>